src/js/components/error.js

```diff
@@ -27,9 +29,18 @@ class ErrorBoundary extends React.Component<Props, { hasError: boolean }> {
       return (
         <EmptyIllustration
           message={
-            <Trans i18nKey="anErrorOccurred">
-              An error occurred. Try the <a href={routes.home()}>home page</a>?
-            </Trans>
+            <>
+              <Trans i18nKey="anErrorOccurred">
+                Unless you pasted a URL incorrectly, it is probably a bug in the
+                code or a networking problem.
+                <br />
+                Our top minds have been alerted.
+                <br />
+                You may need to <a href="./perf">rebuild your query</a> from
+                scratch. Sorry about that.
```

Comment:
This is fine when we only have one React-based page, but I think we'll need a way to provide contextual instructions here eventually.

Reply:
👍